internal/generate/olm-catalog/csv.go

```diff
@@ -271,6 +307,45 @@ func (g BundleGenerator) generateCSV() (fileMap map[string][]byte, err error) {
 	return fileMap, nil
 }
 
+// fillUImetadata populates the CSV with the data obtained from the interactive
+// prompts which appear while generating CSV.
+func fillUImetadata(s InteractiveCSVCmd, csv *olmapiv1alpha1.ClusterServiceVersion) {
```

Comment:
This can be a method on `InteractiveCSVCmd` and live in the file containing the type definition.

Reply:
Modified to have the functions related to interactive command at one place.